payjoin/src/uri/mod.rs

```diff
@@ -128,7 +128,7 @@ impl bitcoin_uri::SerializeParams for &PayjoinExtras {
             .replacen(host, &host.to_uppercase(), 1);
 
         vec![
-            ("pjos", if self.disable_output_substitution { "1" } else { "0" }.to_string()),
+            ("pjos", if self.disable_output_substitution { "0" } else { "1" }.to_string()),
             ("pj", endpoint_str),
         ]
```

Comment:
> Also, when allowing output substition I think we just shouldn't set pjos at all since it's implied to be allowed and would slightly shorten the URI.

Just to be clear the idea would be to simplify this by removing the addition of `pjos=0`?
```suggestion
        if self.disable_output_substitution {
            vec![("pj", endpoint_str)]
        } else {
            vec![("pjos", String::from("1")), ("pj", endpoint_str)]
        }
```

Reply:
Yes but this is backwards - output substitution is implicitly allowed, so we only need to add the `pjos` parameter if output substitution is disabled (i.e. `pjos=0`). We should never have to specify `pjos=1` since that's implicit.